yarn-project/end-to-end/package.json

```diff
@@ -15,12 +15,6 @@
   },
   "jest": {
     "preset": "ts-jest/presets/default-esm",
-    "globals": {
-      "ts-jest": {
-        "useESM": true,
-        "tsconfig": "../tsconfig.json"
-      }
-    },
```

Comment:
The e2e tests started throwing a jest error and it has happened to me before that updating jest resolved so I did that again. With updated jest I had to update the config so that it doesn't throw warning.